test/e2e/cluster_upgrade.go

```diff
@@ -561,13 +554,13 @@ func migRollingUpdateStart(templ string, nt time.Duration) (string, error) {
 	var errLast error
 	var id string
 	prefix, suffix := "Started [", "]."
-	// TODO(mbforbes): Refactor this to use cluster_upgrade.go:retryCmd(...)
 	if err := wait.Poll(poll, singleCallTimeout, func() (bool, error) {
 		// TODO(mbforbes): make this hit the compute API directly instead of
 		//                 shelling out to gcloud.
 		// NOTE(mbforbes): If you are changing this gcloud command, update
 		//                 cluster/gce/upgrade.sh to match this EXACTLY.
-		o, err := exec.Command("gcloud", append(migUdpateCmdBase(),
+		// A `rolling-updates start` call outputs what we want to stderr.
```

Comment:
I think this is as fragile as shelling out to `gcloud` is. Changing it from combined output to standard error does not seem to make it more fragile based on my (admittedly unscientific) knowledge of what `gcloud` writes to stdout vs stderr for a handful of different commands. I do think that we should switch to calling the raw APIs Very Soon&trade;.